### DISCUSSION TITLE
매주 저축금액을 천원단위로 버림처리할 경우 논의할점

### DISCUSSION THREAD

**Discussion by limhyoseok** [DeveloperAcademy-POSTECH/toGather#81] [General]
#67 
위의 논의 결과로 매주 저축금액을 천원단위로 버림 처리하기로 했습니다.

문제는 버림처리로 인해서 전체 저축금액이 변한다는 점인데
초기 선택한 목표금액과 최종 저축금액이 다르다는 것을 사용자가 어떻게 받아 들일까요?
<img width="1087" alt="image" src="https://user-images.githubusercontent.com/59302419/174197024-e51f3172-74ab-41a8-a462-2d506ddceee2.png">

**Comment by Sungwooo** [comment]
저축목표화면과 최종저축내용화면에서 "기간에 따라 목표금액은 바뀔 수 있어요"라는 라이팅을 추가하는게 일단 나을 거 같아요
